server/verify.go

```diff
@@ -293,3 +302,29 @@ func makePool(certs []*x509.Certificate) *x509.CertPool {
 	}
 	return pool
 }
+
+// VerifyTechnology checks Trusted Execution Technology (TEE) attestation reports if the technology
+// supports them, and only then validates that a particular technology has proven that it is in use.
+func VerifyTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTechnology, opts *VerifyOpts) error {
```

Comment:
Let's at least specify GCE, since it relies on the value in the GCE-written firmware event log

Reply:
Done.